resources/qml/PrintSetupSelector/Recommended/RecommendedPrintSetup.qml

```diff
@@ -17,7 +17,9 @@ Item
     property bool settingsEnabled: Cura.ExtruderManager.activeExtruderStackId || extrudersEnabledCount.properties.value == 1
     property real padding: UM.Theme.getSize("default_margin").width
 
-    ColumnLayout
+    function onModeChanged() {}
+
+    Column
```

Comment:
Nooo my `ColumnLayout` 😭 😢 

nah kidding, I get it. Difference in preference 😉

Reply:
I was growing to like the column layout. Then I had to create an animated increasing/decreasing height Rectangle inside it, which might be impossible.